src/anndata/utils.py

```diff
@@ -359,14 +359,14 @@ def warn_once(msg: str, category: type[Warning], stacklevel: int = 1):
 
 def deprecated(
     new_name: str,
-    category: type[Warning] = DeprecationWarning,
+    category: type[Warning] = FutureWarning,
```

Comment:
Maybe we remove this altogether and always force `FutureWarning`?

Reply:
works for me!